tests/SConscript

```diff
@@ -10,6 +10,7 @@ afwdataDir = env.ProductDir("afwdata")
 if afwdataDir:
     env["ENV"]["AFWDATA_DIR"] = afwdataDir
 
-scripts.BasicSConscript.tests(noBuildList=[name + '.cc' for name in pybind11_test_modules])
+scripts.BasicSConscript.tests(noBuildList=[name + '.cc' for name in pybind11_test_modules],
+                              ignoreList=[name + '.py' for name in pybind11_test_modules])
```

Comment:
I'm curious: why is this change needed? I suppose if you use `pytest tests/*.py` instead of `pytest`, as we may have to do initially, that would explain it.